components/ui/components/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<resources
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+<!-- UI button styling -->
+<style name="NeutralButton" parent="Widget.MaterialComponents.Button.TextButton">
```

Comment:
Nice! Great so see that a style seems to be sufficient. Does it work nicely in Fenix?